apis/v1alpha1/types.go

```diff
@@ -17,9 +17,8 @@ limitations under the License.
 package v1alpha1
 
 import (
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
 	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
The `metav1` import should be in a group above `crossplane/*` imports. I'll check to see why linter didn't catch this.

Reply:
+1. I tried to manually switch the order, the linter threw a warning.